scripts/autoloaders/state.gd

```diff
@@ -1,7 +1,7 @@
 extends Node
 
 var max_power = 100.0
-var power = 100.0
+var power = 0.0
```

Comment:
I've made some changes to this, so the default now is 0, and it also gets reset to 0 when the game starts.